packages/cursorless-org-docs/src/css/custom.css

```diff
@@ -28,3 +28,19 @@
 .hidden {
   display: none;
 }
+
+.card {
```

Comment:
As far as I could tell we don't really have a css library for the docs page? I didn't want to add one for this simple use case.